### DISCUSSION TITLE
Welcome to traviso.js Discussions!

### DISCUSSION THREAD

**Discussion by axaq** [axaq/traviso.js#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zinecreati** [comment]
Is this framework still being supported?

**Reply by axaq** [reply]
Hi there. No I wasn't actively updating it for a long time unfortunately due to my daily job weight and life in general :). But now with the help of coding agents I started progressing and updating much more quickly. If you are still interested, let me know what you want to see next from traviso.js. I have one release coming for pixi.js v7 support and then another one with pixi v8 support to modernise the underlying code. After that I am going to focus on adding new features.
